lib/matplotlib/widgets.py

```diff
@@ -1007,7 +1007,12 @@ def __init__(self, ax, labels, active=0, activecolor='blue'):
             The index of the initially selected button.
         activecolor : color
             The color of the selected button.
+        orientation : str
+            The orientation of the buttons: 'vertical' (default), or 'horizontal'.
```

Comment:
The keyword being a bit confusing here, I think it's worth detailing a bit more what orientation means (which is the set of radio buttons are on the same horizontal axis or vertical axis).